src/js/common/schemaform/FormSaved.jsx

```diff
@@ -7,9 +7,19 @@ import Scroll from 'react-scroll';
 
 import { focusElement } from '../utils/helpers';
 import { fetchInProgressForm, removeInProgressForm } from './save-load-actions';
+import { handleVerify } from '../../common/helpers/login-helpers.js';
 
 import FormStartControls from './FormStartControls';
 
+moment.updateLocale('en', {
```

Comment:
@cvalarida @jbalboni thanks for any insights on whether there is a better place to locate this.